java/src/jmri/jmrix/loconet/messageinterp/LocoNetMessageInterpret.java

```diff
@@ -4253,6 +4265,10 @@ private static String interpretOpcImmPacket(LocoNetMessage l) {
                                     // GG=00 Reserved for future use
                                     log.debug("CV # {}, Reserved (GG=0); {}", cvnum, packetInt[4]);    
                             }
+                        } else if (packetInt.length == 3) {
```

Comment:
@devel-bobm The draft standards Mar 9 2023 are proposing byte 3 be all 8 bits as an aspect, and using only length to identify the packet, and since currently there is no 3 byte Extended accessory packet that isnt an aspect setter I thought we could just do it now.